.github/workflows/test.yml

```diff
@@ -61,19 +64,17 @@ jobs:
         shell: bash
         # NumPy is installed first because the pygdal build needs it.
         run: |
-          pip install --disable-pip-version-check --upgrade pip setuptools wheel uv
-          uv pip install --system numpy${{ matrix.numpy }}
+          uv pip install --system setuptools numpy${{ matrix.numpy }}
           uv pip install --system ${{ matrix.gdal }} --no-build-isolation
           uv pip install --system -e .[test,cityhash] ${{ matrix.pins }}
           uv pip list
 
       - name: Run tests
         shell: bash
-        run: |
-          uv run pytest dask_geomodeling
+        run: pytest dask_geomodeling
```

Comment:
Funny thing: we had `uv pip install --system`, and that created a neat set of packages on the system python. Then `uv run` just created a whole new environment, with latest versions, and runs the tests there. So this was actually a big mistake.